dbt/models/default/default.vw_pin_appeal.sql

```diff
@@ -1,10 +1,20 @@
 -- View containing appeals by PIN
 
--- This CTE exists only so that we can join reason descriptions onto cleaned
--- reason codes.
+-- CTE so that we can join reason descriptions onto cleaned reason codes and
+-- drop some dupes from htpar.
 WITH reasons AS (
-    SELECT
-        htpar.*,
+    SELECT DISTINCT
+        htpar.parid,
+        htpar.taxyr,
+        htpar.caseno,
+        htpar.user38,
+        htpar.user104,
+        htpar.resact,
+        htpar.hrstatus,
+        htpar.cpatty,
+        -- user125 isn't used in the view, but it's FilingID and we want to
+        -- include it to make sure we're only dropping true dupes using DISTINCT
+        htpar.user125,
```

Comment:
It's probably unnecessary, but Mirella implied that filing id is what she would use to determine if something has been published twice. This is just me being extra sure, rather than as parsimonious as possible.